### DISCUSSION TITLE
Spatie Media Library conversions

### DISCUSSION THREAD

**Discussion by habib19** [filamentphp/filament#5472] [Help] [answered]
I want create  thumbnails version of my Product image; so i add the following code to my product model:
public function registerMediaConversions(Media $media = null): void
    {
        $this->addMediaConversion('thumb')
              ->width(368)
              ->height(232);
    }
Now, how to get the thumb image

**Comment by ArtMin96** [comment, marked as answer]
[https://filamentphp.com/docs/2.x/spatie-laravel-media-library-plugin/table-columns#using-conversions](https://filamentphp.com/docs/2.x/spatie-laravel-media-library-plugin/table-columns#using-conversions)

**Reply by habib19** [reply]
I'm so grateful...
